### PR TITLE
Add treemaker

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/treemaker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - treemaker = treemaker:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```
Comment (mbargull) [inline recipes/treemaker/meta.yaml]: Not yet on cb3 even if `skeleton` thinks otherwise 😉

File: recipes/treemaker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - treemaker
+  commands:
+    - treemaker --help
+
+about:
+  home: https://github.com/SimonGreenhill/treemaker
+  license: BSD License
```
Comment (mbargull) [inline recipes/treemaker/meta.yaml]:
```yaml
  license: BSD-3-Clause
  license_file: LICENSE
```

File: recipes/treemaker/meta.yaml (hunk 1 above)
Comment (mbargull) [reply recipes/treemaker/meta.yaml]: I.e., need `build:` instead.

Comment (druvus) [reply recipes/treemaker/meta.yaml]: thanks, bad habit of trying to fix a PR before running out of office.

**Comment by mbargull** [comment]
Nice!

**Pull request merged**